### PR TITLE
chore: update dependencies

### PR DESCRIPTION
**Pull request by Zidious**
We had a bit to many dependabot PR's, updated all here so next month we hopefully won't have so many PR's. 

- This PR updates dependencies and rebuilds all actions as they rely some of the updated dependencies, this will have to occur manually, but https://github.com/dequelabs/axe-api-team-public/issues/57 is open to address this issue

No QA Required

### CODE REVIEW

File: .github/actions/label-and-move-released-issues-v1/src/run.test.ts
Comment (Zidious) [inline .github/actions/label-and-move-released-issues-v1/src/run.test.ts:257]: As of `@actons/github` v6, they changed the way the generate the GET endpoints which completely broke the way we were calling them, they appear to be proxying old + new endpoints with a new `handler` function which attempted to destructure a variable which ended being undefined: 

>       TypeError: Cannot destructure property ‘decorations’ of ‘endpointMethodsMap.get(...).get(...)’ as it is undefined.



Similar, not exact issue: https://github.com/octokit/plugin-rest-endpoint-methods.js/issues/668


I'll be opening a issue in that repo, but for now this a work around

Comment (Zidious) [reply .github/actions/label-and-move-released-issues-v1/src/run.test.ts:257]: Opened an issue to see if this was an intended change as of v6: https://github.com/octokit/plugin-rest-endpoint-methods.js/issues/683

**Comment by stephenmathieson** [comment]
I do not understand why this repository contains 26+ files. This seems like a huge amount of work to maintain

**Comment by Zidious** [comment]
> I do not understand why this repository contains 26+ files. This seems like a huge amount of work to maintain

As some of the dependencies get used in the other actions, I've had to manually rebuild the `dist` for each action. This should be addressed in https://github.com/dequelabs/axe-api-team-public/issues/57 which will automate everything. In this I am manually updating deps, this is automated also (when it runs on the 1st of every month)

File: .github/actions/label-and-move-released-issues-v1/src/run.test.ts
Comment (Zidious) [reply .github/actions/label-and-move-released-issues-v1/src/run.test.ts:257]: This issue was fixed by GitHub but the fix needs to be upstreamed to `@actions/github` before I can pull in the change. Will wait for that before merging, moving this to draft.

Comment (Zidious) [reply .github/actions/label-and-move-released-issues-v1/src/run.test.ts:257]: I was waiting for the above PR to be merged but it's not moving. 


Will un-draft this PR and get this moving, opened issue to revert the work around once there has been some movement: https://github.com/dequelabs/axe-api-team-public/issues/80

**Comment by Zidious** [comment]
Un-drafting this PR, please see: https://github.com/dequelabs/axe-api-team-public/pull/58#discussion_r1380166517

**Review by straker — APPROVED** [review]

**Pull request merged**
